fabtests/pytest/common.py

```diff
@@ -68,23 +68,23 @@ def run(self):
 class ClientServerTest:
 
     def __init__(self, cmdline_args, executable, iteration_type=None, completion_type="transmit_complete",
-                 prefix_type="wout_prefix", datacheck_type="wout_datacheck", message_size=None,
+                 prefix_type="wout_prefix", datacheck_type="wout_datacheck", message_size_range=None,
```

Comment:
I think this argument probably should be named `message_size_expression` or `message_size_pattern` because it can take 3 kinds of input: a number, the word "all", and a range like "r:bgn,interval,end".

Also, can you add a help text section for this function with this info?

Reply:
I would say the name change is unnecessary.

Reply:
That looks good.

Reply:
The title of the PR should be updated, too.

Reply:
Done